documentation/modules/ROOT/pages/transformations/timezone-converter.adoc

```diff
@@ -0,0 +1,181 @@
+
+[[timezone-converter-usage]]
+
+[[basic-example-timezone-converter]]
+== Example: Basic configuration
+
+To convert the time based fields in an event record to a target timezone, configure the `TimezoneConverter` SMT in the Kafka Connect configuration for a connector.
+For example, to convert the all timestamp fields in an event record from UTC to the `Pacific/Easter` timezone:
+
+[source]
+----
+transforms=convertTimezone
+transforms.convertTimezone.type=io.debezium.transforms.TimezoneConverter
+transforms.convertTimezone.converted.timezone=Pacific/Easter
+----
+
+The following example shows how to convert the all timestamp fields in an event record:
+
+.Effect of applying the `TimezoneConverter` SMT
+====
+Event record before it is processed by the `TimezoneConverter` transformation::
+
+Value before the SMT processes the event record:::
++
+[source, json]
+----
+{
+        "before": null,
+        "after": {
+            "id": 1,
+            "first_name": "Anne",
+            "last_name": "Kretchmar",
+            "email": "annek@noanswer.org",
+            "created_at": "2011-01-11T16:40:30.123456789+00:00"
+        },
+        "source": {
+            "version": "2.4.0.Aplha2",
+            "connector": "postgresql",
+            "name": "PostgreSQL_server",
+            "ts_ms": 1559033904863,
+            "snapshot": true,
+            "db": "postgres",
+            "sequence": "[\"24023119\",\"24023128\"]",
+            "schema": "public",
+            "table": "customers",
+            "txId": 555,
+            "lsn": 24023128,
+            "xmin": null
+        },
+        "op": "c",
+        "ts_ms": 1559033904863
+    }
+----
+
+Event record after it is processed by the `TimezoneConverter` transformation::
+
+Value after the SMT converts the timestamp field to the target timezone:::
++
+[source, json]
+----
+{
+        "before": null,
+        "after": {
+            "id": 1,
+            "first_name": "Anne",
+            "last_name": "Kretchmar",
+            "email": "annek@noanswer.org",
+            "created_at": "2011-01-11T11:40:30.123456789-05:00"
+        },
+        "source": {
+            "version": "2.4.0.Aplha2",
+            "connector": "postgresql",
+            "name": "PostgreSQL_server",
+            "ts_ms": 1559033904863,
+            "snapshot": true,
+            "db": "postgres",
+            "sequence": "[\"24023119\",\"24023128\"]",
+            "schema": "public",
+            "table": "customers",
+            "txId": 555,
+            "lsn": 24023128,
+            "xmin": null,
+            "id": 100
+        },
+        "op": "c",
+        "ts_ms": 1559033904863,
+        "event_timestamp": 1626102708861
+    }
+----
+====
+
+[[advanced-example-timezone-converter]]
+== Example: Advanced configuration
+
+For example, to convert the `created_at`, and `updated_at` timestamp fields in an event record from UTC to a fixed offset of `+05:30`, configure the `include.fields` configuration option as follows:
```

Comment:
```suggestion
Instead of converting all  timestamp fields in an event record, you can configure the SMT to convert specific fields only. 
The following example shows how you might use the `include.fields` option in the SMT configuration to add convert only the `created_at`, and `updated_at` timestamp fields in an event record.
Instead of specifying a geographic timezone designator,  this configuration uses a fixed offset to convert the  time from UTC to `+05:30`.


to convert all timestamp fields in an event record from UTC to the `Pacific/Easter` timezone, add the following lines to your connector configuration:
```

Reply:
> to convert all timestamp fields in an event record from UTC to the `Pacific/Easter` timezone, add the following lines to your connector configuration:

If my understanding is accurate, this portion appears to be unrelated to the preceding example and seems to have been included in the suggested revisions by mistake.

Reply:
Yes, @ani-sha, you are correct. Seems that I mistakenly pasted this duplicate fragment from part of an earlier sentence.  Apologies for causing any confusion!